x-pack/plugins/case/server/services/index.ts

```diff
@@ -45,20 +63,50 @@ interface PushedArgs {
 }
 
 interface GetCaseArgs extends ClientArgs {
-  caseId: string;
+  id: string;
 }
 
 interface GetCasesArgs extends ClientArgs {
   caseIds: string[];
 }
 
-interface FindCommentsArgs extends GetCaseArgs {
+interface GetSubCasesArgs extends ClientArgs {
+  ids: string[];
+}
+
+interface FindCommentsArgs {
```

Comment:
nit:
```suggestion
interface FindCommentsArgs {
```
interface FindCommentsArgs extends ClientArgs {